Add monitoring and minor copy update (#1013)
hashicorp/terraform-website: content/source/docs/enterprise/before-installing/reference-architecture/aws.md

for the defined bucket and all object storage requests are routed to the
highly available infrastructure supporting S3.

### Monitoring

> While there is not currently a full monitoring guide for Terraform Enterprise, information around

Reviewer: ```suggestion
There is not currently a full monitoring guide for Terraform Enterprise. The following pages include information relevant to monitoring:

```

> [logging](../../admin/logging.html),

Reviewer: ```suggestion
- [Logging](../../admin/logging.html),
```

> [diagnostics](../../support/index.html)

Reviewer: ```suggestion
- [Diagnostics](../../support/index.html)
```

> as well as [reliability and

Reviewer: ```suggestion
```

> availability](../../system-overview/reliability-availability.html)

Reviewer: ```suggestion
- [Reliability and Availability](../../system-overview/reliability-availability.html)
```

> can be found on our website.

Reviewer: ```suggestion
```
